src/server/api/schema.js

```diff
@@ -649,23 +650,71 @@ const rootMutations = {
         return userData;
       }
     },
-    resetUserPassword: async (_, { organizationId, userId }, { user }) => {
+    resetUserPassword: async (
+      _,
+      { organizationId, userId },
+      { user, loaders }
+    ) => {
       if (user.id === userId) {
         throw new Error("You can't reset your own password.");
       }
+      // might need to impliment user password change available elsewhere.
+
       await accessRequired(user, organizationId, "ADMIN", true);
 
-      // Add date at the end in case user record is modified after password is reset
-      const passwordResetHash = uuidv4();
-      const auth0_id = `reset|${passwordResetHash}|${Date.now()}`;
+      const organization = await loaders.organization.load(organizationId);
 
-      const userRes = await r
-        .knex("user")
-        .where("id", userId)
-        .update({
-          auth0_id
+      const passportStrategy = getConfig("PASSPORT_STRATEGY", organization);
+      if (passportStrategy === "auth0") {
+        // BEGINNING
+        const { email } = await r
+          .knex("user")
+          .select("email")
+          .where({
+            "user.id": userId
+          })
+          .first();
+
+        const auth0Domain = getConfig("AUTH0_DOMAIN", organization);
+        const auth0ClientID = getConfig("AUTH0_CLIENT_ID", organization);
+        const url = `https://${auth0Domain}/dbconnections/change_password`;
+        const body = JSON.stringify({
+          client_id: auth0ClientID,
+          email,
+          connection: "Username-Password-Authentication"
         });
-      return passwordResetHash;
+        try {
+          let res = await httpRequest(url, {
+            method: "POST",
+            retries: 2,
+            timeout: 5000,
+            headers: { "Content-Type": "application/json" },
+            body,
+            validStatuses: [200],
+            compress: false
+          });
+          res = await res.text();
+          console.log(res, email);
+          return res;
+        } catch (err) {
+          //handles error and sends it to the client
+          throw new Error(err);
+        }
+      } else {
+        //returns one of the options
```

Comment:
Can you expand more on this comment?

Reply:
I believe this is psuedo code that I failed to remove.  Thanks for catching that!